runtime/autoload/provider/clipboard.vim

```diff
@@ -175,6 +182,10 @@ function! s:clipboard.set(lines, regtype, reg) abort
 
   if s:cache_enabled == 0
     call s:try_cmd(s:copy[a:reg], a:lines)
+    "Cache it anyway we can compare it later to get regtype of the yank
+    let s:selections[a:reg] = copy(s:selection)
+    let s:selections[a:reg].data = [a:lines, a:regtype]
+    let s:selections[a:reg].stderr_buffered = v:false
```

Comment:
this should not be changed.

Reply:
Why ?

So should the default value of `cache_enabled` be `1` . `pbcopy` on mac has it set to `0` . same for default `g:clipboard` . Without this those won't work . Also I though `cache_enable` meant the provider will be kept alive in daemon mode to hold ownership of clipboard. Isn't that X11 specific ? If the provider doesn't have a  daemon mode or the idea of clipboard ownership as X11 won't it be weird to have `cache_enabled = 1`

Can you please elaborate on what you're proposing as change ?

Reply:
what has any of that to do with this line?

Reply:
Oh did you mean line 188 only ?

Reply:
yes, otherwise I would have referred to some other line[s].

Reply:
(to be fair, I wasn't 100% either; the github code comment UI is not very clear about whether you talk about a single line or the whole quoted code block.)

Reply:
Yes, that's fair, but for future reference: commenting on the **beginning** of a block is often ambiguous with the entire block, though I like to think a line being quoted in the middle of a block refers to that line only.